### DISCUSSION TITLE
mod.lua help

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#1169] [Q&A] [answered]
can someone tell me how to make arrows move or jump using mod.lua it would really help :)

**Comment by Ibexlol** [comment, marked as answer]
watch this https://www.youtube.com/watch?v=9c1ufokuQ28&t=334s

**Reply by isakube** [reply]
Correction: [this](https://github.com/KadeDev/Kade-Engine/wiki)

**Reply by glowsoony** [reply]
thanks

**Comment by isakube** [comment]
https://github.com/KadeDev/Kade-Engine/wiki
